content/en/docs/v3.6/feature-gates/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Feature Gates
+weight: 10
```

Comment:
In the deploy preview Feature Gates appears as the first item in the bullet list for v3.6 docs:

![image](https://github.com/user-attachments/assets/8034823c-4406-45c1-adb8-64255c82ae0f)

I think FeatureGates should be ranked lower in the list, after `Install` bullet point would make sense to me.

```suggestion
weight: 1175
```